src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -827,16 +827,7 @@ protected override void DecodeWellKnownAttributeImpl(ref DecodeWellKnownAttribut
 
         private bool IsValidUnscopedRefAttributeTarget()
         {
-            switch (RefKind)
-            {
-                case RefKind.Out:
-                    return true;
-                case RefKind.Ref:
-                    var type = Type;
-                    return type is null || type.IsErrorTypeOrRefLikeType();
-                default:
-                    return false;
-            }
+            return ParameterHelpers.IsRefScopedByDefault(this);
```

Comment:
nit: consider inlining.

Reply:
I'll leave this as-is since we have `IsValidUnscopedRefAttributeTarget()` instance methods in several `Symbol` implementations so this is consistent.